src/ledger.ts

```diff
@@ -77,19 +75,19 @@ export class LedgerCanister {
    * Transfer ICP from the caller to the destination `accountIdentifier`.
    * Returns the index of the block containing the tx if it was successful.
    *
-   * TODO: support remaining options (subaccounts, memos, etc.)
-   *
    * @throws {@link TransferError}
    */
   public transfer = async ({
     to,
     amount,
     memo,
+    fee,
```

Comment:
This feels weird to me. Why do we need it?

Reply:
What feels weird to you?!!?

`ts` currently productive definition -> https://github.com/dfinity/nns-dapp/blob/ee681ffbda016edb4b006e3881192e9084de8c36/frontend/ts/src/canisters/ledger/model.ts#L13

Reply:
so I guess it's correct to have fee and memo but again, let me know what feels weird to you

Reply:
It feels weird because the problems we had splitting neurons was that the transaction fee passed from the Governance Canister, to the Ledger Canister did not match the transaction fee there.

But if it works, then no problem.

Which is the functionality in the NNS Dapp that needs the fee? Or you are just adding the functionality?

Reply:
I see, thanks for the explanation.

Transaction as in ts for Flutter. Reference: https://github.com/dfinity/nns-dapp/blob/ee681ffbda016edb4b006e3881192e9084de8c36/frontend/ts/src/canisters/ledger/RequestConverters.ts#L43

Reply:
Yes.